test/libsolidity/ASTJSON/assembly/nested_functions.json

```diff
@@ -16,7 +16,6 @@
       "baseContracts": [],
       "contractDependencies": [],
       "contractKind": "contract",
-      "fullyImplemented": true,
```

Comment:
Why are these removed?

Reply:
because nested_functions.sol doesn't successfully compile
it does however successfully output an AST tree
but due to the error the analysis is interrupted